test/e2e/network/netpol/network_policy.go

```diff
@@ -130,10 +130,11 @@ var _ = common.SIGDescribe("Netpol", func() {
 		})
 
 		ginkgo.It("should support a 'default-deny-ingress' policy [Feature:NetworkPolicy]", func() {
-			nsX, _, _, k8s := getK8sNamespaces(f)
+			_, _, _, k8s := getK8sNamespaces(f)
```

Comment:
Every caller to `getK8sNamespaces` now ignores the first three return values. If the method is going to stay the way it is, you should just remove those return values. But it would be better to fix things so that it knows the real namespace name again.